tap_github/__init__.py

```diff
@@ -211,7 +214,50 @@ def get_catalog():
 
     return {'streams': streams}
 
-def do_discover():
+def verify_repo_access(url_for_repo, repo):
+    try:
+        authed_get("verifying repo access", url_for_repo)
+    except NotFoundException as e:
+        logger.error("API token does not have the permission to access '%s' repository.", repo)
+        raise e
+    except BadCredentialsException as e:
+        logger.error("API token is invalid. Please enter correct credentials.")
+        raise e
+
+def verify_org_access(url_for_org, org):
+    try:
+        authed_get("verifying org access", url_for_org)
+    except NotFoundException as e:
+        logger.error("'%s' is not an Oragnization.", org)
```

Comment:
@harshpatel4crest There is a typo in the word Organization

Reply:
Ok, I will solve that